src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -56,207 +81,331 @@ def __init__(self,
         self._details = details
         self._initial_metadata = initial_metadata
         self._trailing_metadata = trailing_metadata
+        self._debug_error_string = debug_error_string
 
     def code(self) -> grpc.StatusCode:
-        """
+        """Accesses the status code sent by the server.
+
         Returns:
           The `grpc.StatusCode` status code.
         """
         return self._code
 
     def details(self) -> Optional[str]:
-        """
+        """Accesses the details sent by the server.
+
         Returns:
           The description of the error.
         """
         return self._details
 
     def initial_metadata(self) -> Optional[Dict]:
```

Comment:
AioRpcError does not implement `grpc.aio.Call`, see `class AioRpcError(grpc.RpcError):`. I'm torn about this topic, because on the one hand, I hope the API to be consistent as you do, on the other hand, those informations are ready because this is the final state so there is no need for `async def`. WDYT?

Reply:
The case that I worry about is someone who is used to, e.g., using `await call.metadata()` does `await error.metadata()`. The error would like like

```
TypeError: object dict can't be used in 'await' expression
```
I suppose this is a pretty clear indication that you shouldn't be calling `await`.